ui/tempo-plugin/src/model/types.ts

```diff
@@ -0,0 +1,35 @@
+import { Trace, Spans } from '@perses-dev/core';
```

Comment:
if you run `make fixlicense` it will put the license in every files where it is missing

Reply:
I ran `make fixlicense`, thanks for the tip!